### PR TITLE
Add a `--colorScheme` param to `new-tab`, `split-pane`

### PR DESCRIPTION
**Pull request by zadjii-msft**
This is entirely self-serving. In my go-to config, I like having some of
the panes for a given profile in a different color scheme. This will let
a user pass `--colorScheme <scheme name>` to manually override the
scheme for that profile. Neat!

### CODE REVIEW

File: src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp [hunk 1]
```diff
@@ -651,6 +660,31 @@ namespace TerminalAppLocalTests
             VERIFY_ARE_EQUAL(til::color(myArgs.TerminalArgs().TabColor().Value()), expectedColor);
             VERIFY_IS_NULL(myArgs.TerminalArgs().ProfileIndex());
             VERIFY_IS_TRUE(myArgs.TerminalArgs().Profile().empty());
+            VERIFY_IS_TRUE(myArgs.TerminalArgs().ColorScheme().empty());
+        }
+        {
+            AppCommandlineArgs appArgs{};
+            std::vector<const wchar_t*> rawCommands{ L"wt.exe", subcommand, L"--colorScheme", L"Monokai" };
```
Comment (WSLUser) [inline src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp:667]: Looks like you've hard coded "Monokai" for the color scheme. This should be a empty string looking for input from the user.

Comment (zadjii-msft) [reply src/cascadia/LocalTests_TerminalApp/CommandlineTest.cpp]: It's a test, of course I hardcoded the string...

**Review by carlos-zamora — APPROVED** [review]

File: src/cascadia/TerminalSettingsModel/ActionArgs.cpp [hunk 2]
```diff
@@ -66,6 +66,10 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             const til::color tabColor{ _TabColor.Value() };
             ss << fmt::format(L"tabColor: {}, ", tabColor.ToHexString(true));
         }
+        if (!_ColorScheme.empty())
+        {
+            ss << fmt::format(L"colorScheme: {}, ", _ColorScheme);
```
Comment (WSLUser) [inline src/cascadia/TerminalSettingsModel/ActionArgs.cpp:71]: Might it make sense to use FMT_COMPILE here  (and the one below) instead similar to https://github.com/microsoft/terminal/pull/9581 or would it make more sense to do all the params at once in a separate PR?

Comment (zadjii-msft) [reply src/cascadia/TerminalSettingsModel/ActionArgs.cpp:71]: Probably makes sense to do them all in a single pass IMO

File: src/cascadia/TerminalSettingsModel/ActionArgs.h [hunk 3]
```diff
@@ -68,6 +68,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         WINRT_PROPERTY(Windows::Foundation::IReference<int32_t>, ProfileIndex, nullptr);
         WINRT_PROPERTY(winrt::hstring, Profile, L"");
         WINRT_PROPERTY(Windows::Foundation::IReference<bool>, SuppressApplicationTitle, nullptr);
+        WINRT_PROPERTY(winrt::hstring, ColorScheme, L"");
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsModel/ActionArgs.h:71]: no need for the `L""` here (actually, not above either.)

File: src/cascadia/TerminalSettingsModel/TerminalSettings.cpp [hunk 4]
```diff
@@ -118,6 +118,14 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
             {
                 settings.SuppressApplicationTitle(newTerminalArgs.SuppressApplicationTitle().Value());
             }
+            if (!newTerminalArgs.ColorScheme().empty())
+            {
+                const auto schemes = appSettings.GlobalSettings().ColorSchemes();
+                if (const auto scheme = schemes.TryLookup(newTerminalArgs.ColorScheme()))
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsModel/TerminalSettings.cpp:124]:
```suggestion
                if (const auto& scheme = schemes.TryLookup(newTerminalArgs.ColorScheme()))
```

Who knows, the compiler may optimize this one. Probably won't optimize the above one.

**Review by DHowett — APPROVED** [review]
My notes should have come with an approval.

**Review by DHowett — APPROVED** [review]

**Pull request merged**